src/clusterfuzz/_internal/tests/core/bot/tasks/trials_test.py

```diff
@@ -40,6 +42,22 @@ def setUp(self):
 
     test_helpers.patch(self, ['random.random'])
 
+  def source_side_test(self, config_file_content, probability, app_name,
+                       app_args, trial_app_args):
+    """source side trials test template"""
+    test_utils.set_up_pyfakefs(self)
```

Comment:
I suggest to move this line to the setUp method as well, otherwise the fake file system might not work. Since SUT calls os.path.exists. The line below should stay here, as the config file only exists in the source side tests.

Reply:
done